{"uuid": "51d2e0f1-7685-498e-b1ae-99a5b340333b", "children": ["bc9417d3-36ff-4ac5-948a-c44a57a775e4", "0c87a3e1-994c-4a70-a424-320edb6997b5", "6fbaa61e-107f-40f4-a701-856ea6474493", "5ecb39c3-8095-4af7-8432-9cc6a8273567", "55a9cc99-67b5-4848-aa83-6a6bf53cbda8"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1586839776297, "stop": 1586839776297}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "NameError: name 'mail_pass' is not defined\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 707, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/sonic/Project/Python/Pytest/scripts/test_case.py\", line 41, in teardown_class\n    EmailHandler().send_email()\n  File \"/Users/sonic/Project/Python/Pytest/uti/EmailHandler.py\", line 59, in send_email\n    smtp_obj.login(mail_user, mail_pass)\n"}, "start": 1586839777428, "stop": 1586839788000}], "start": 1586839776297, "stop": 1586839788000}